stock_maturity/views/stock_lot_views.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding='UTF-8' ?>
+<odoo>
+    <record id="view_production_lot_form" model="ir.ui.view">
+        <field name="name">stock.production.lot.inherit.form</field>
+        <field name="model">stock.lot</field>
+        <field name="inherit_id" ref="stock.view_production_lot_form" />
+        <field name="arch" type="xml">
+            <field name="ref" position="after">
+                <field name="maturity_date" />
+            </field>
+            <xpath expr="//div[hasclass('oe_title')]" position="inside">
+                <span
+                    class="badge text-bg-warning"
+                    invisible="not product_maturity_alert"
+                >Not mature yet</span>
+            </xpath>
```

Comment:
Will be easy to show an alert (alert-warning) on the top of the form.
Otherwise, fix bottom margin (I think would be too much space to show this kind of badge)

<img width="480" height="300" alt="Image" src="https://github.com/user-attachments/assets/668047f5-f21b-4898-bf6b-005c21ee8f5d" />